lib/pdfkit/html_preprocessor.rb

```diff
@@ -0,0 +1,23 @@
+class PDFKit
+  class HtmlPreprocessor
```

Comment:
If anyone ever decides to pull the stylesheet stuff into `HTMLPreprocessor`, maybe it'll have enough heft to be its own class. It'll be easy enough to change back at that point.